drivers/bcmxcp.c

```diff
@@ -1207,7 +1207,7 @@ void init_limit(void)
 
 	/* Horn Status: */
 	value = answer[BCMXCP_EXT_LIMITS_BLOCK_HORN_STATUS];
-	if (value >= 0 && value <= 2) {
+	if (value <= 2) {
```

Comment:
Note: currently `answer[]` is an array of unsigned values, so compilers warn about check for `... >= 0` as redundant.